src/core/ColorPicker/ColorBuilder.tsx

```diff
@@ -96,10 +109,26 @@ export const ColorBuilder = React.forwardRef(
           h: hue,
           s: hsvColor.s,
           v: hsvColor.v,
+          a: hsvColor.a,
+        };
+        applyHsvColorChange(newHsvColor, selectionChanged);
+      },
+      [applyHsvColorChange, hsvColor],
+    );
+
+    // Update opacity slider change
+    const updateOpacitySlider = React.useCallback(
```

Comment:
If this one is called `updateOpacitySlider`, maybe rename the other one to `updateHueSlider`

Reply:
Renamed